public-docs/community-resources.md

```diff
@@ -0,0 +1,41 @@
+---
+id: community-resources
+title: Community Resources
+---
+
+This page is here to guide you through the Reaction ecosystem. As we've stated before, we wouldn't be here without our very engaged community. Here we'd like to introduce you to some of the best things we've seen out there.
```

Comment:
I'd prefer to editorialize a bit less here, but it works. Could consider something like this:

> There's a very active community building retail experiences with Reaction Commerce. This page serves as a list of active open-source community projects.

Reply:
This sentence was actually straight from the old Community Resources page. You're right though, your version is much better!